WordPress/src/main/res/layout/reader_recommended_blog_item.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/reader_interest_container"
```

Comment:
Seems like a copy-paste mistake 😊. Can we rename it to represent recommended blog container or remove if we don't need it?

Reply:
ohh yes we can remove it, good catch! 👍